test/test-unbind.js

```diff
@@ -25,7 +25,7 @@ conn.once('ready', function () {
     later(function(){
       // This will emit a NOT_FOUND error b/c we're no longer bound to the exchange
       var thrown = false;
-      conn.addListener('error', function(e){
+      exchange.addListener('error', function(e){
```

Comment:
Need some help on this one: the error event is not triggered on connection for some reason. That is not the case if I don't add error handlers on `channelOpenOk`.